unlock-app/src/utils/validators.js

```diff
@@ -1,9 +1,14 @@
 // tests whether a field's value was not entered by the user
 export const isNotEmpty = val => val || val === 0
 
-// tests whether a number is postive and not a decimal number
-export const isPositiveInteger = val =>
-  !isNaN(val) && +val === parseInt(val) && +val > 0
+// tests whether a number is positive (0 included) and not a decimal number
+export const isPositiveInteger = val => {
+  const parsedInt = parseInt(val)
+  return !isNaN(parsedInt) && val == parsedInt && +val >= 0
```

Comment:
This is probably some js knowledge that I am lacking,  but what does `+val >= 0` provide over `val >= 0`